platform/ui/src/components/Toolbox/ToolboxUI.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { PanelSection, ToolSettings, Tooltip } from '../../components';
+import classnames from 'classnames';
+
+const ItemsPerRow = 4;
+
+function ToolboxUI(props) {
```

Comment:
Need 1-2 sentences describing what a ToolboxUI is, and why it isn't just Toolbox itself (which is also a tsx file)